src/components/TokenLocking/Leaderboard.tsx

```diff
@@ -164,20 +169,24 @@ const LeaderboardPage = ({ index, onLoadMore }: { index: number; onLoadMore?: ()
   }
   return (
     <>
-      {rows.map((row) => (
-        <StyledTableRow key={row.holder}>
-          <StyledTableCell align="center">
-            <Ranking position={row.position} />
-          </StyledTableCell>
-          <StyledTableCell align="left">
-            <LookupAddress address={row.holder} />
-          </StyledTableCell>
-          <StyledTableCell align="left">{formatAmount(formatEther(row.lockedAmount), 0)}</StyledTableCell>
-        </StyledTableRow>
-      ))}
+      {rows.map((row) => {
+        return row.position === ownEntry?.position ? (
```

Comment:
I think comparing the `holder` field should be the securer approach. 
Not sure if the position could lead to problems if the positions update in between two requests or something similar